be1-go/cli/organizer/organizer.go

```diff
@@ -44,36 +44,51 @@ func Serve(context *cli.Context) error {
 	done := make(chan struct{})
 	go h.Start(done)
 
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-		serveWs(h, w, r)
+	go orgCreateAndServeWs(hub.WitnessSocketType, h, port)
```

Comment:
I'd suggest having two different ports - one for communication with clients and another for communication with witnesses. This would allow easier debugging of traffic by just monitoring one of the ports than filtering based on the path